docker-compose.yml

```diff
@@ -1,8 +1,10 @@
 db:
   build: db
```

Comment:
Ok, I can't test it with version 2 at the moment, so I'll just remove the `build` line.